src/device-registry/utils/create-event.js

```diff
@@ -84,8 +77,8 @@ const createEvent = {
           constants.DATAWAREHOUSE_METADATA
         }.sites\`.id = \`${table}\`.site_id 
         WHERE timestamp 
-       >= "${start ? formatDate(start) : twoMonthsBack}" AND timestamp <= "${
-        end ? formatDate(end) : currentDate
+       >= "${start ? start : twoMonthsBack}" AND timestamp <= "${
```

Comment:
just removed the date formatting which could be potential error source when querying data.